community.main/pom.xml

```diff
@@ -23,24 +23,9 @@
         <developerConnection>${project.scm.connection}</developerConnection>
     </scm>
 
-    <dependencies>
-        <dependency>
-            <groupId>org.kaazing</groupId>
-            <artifactId>community.license</artifactId>
-            <!-- We really want -->
-            <!--     <version>{project.version}</version> -->
-            <!-- but due to inheritance rules, this won't work. -->
-            <version>develop-SNAPSHOT</version>
```

Comment:
Note the comment here.  See where you do the version now for the follow-up comment.  (This is still required.)